citi-onboarding/CESC-client
# (Feature/our_partners): create the section our partners

#What I did
Created the section our partners
#How to test
Open the terminal and go to this project folder

At the terminal:

git checkout development
git pull
git checkout feature / our partners
git pull origin feature / our partners
yarn install
yarn start


On the open pages of VScode - our partners

Test the things you think are worth testing, even the ones that aren't in this description

make sure the sizes and positions agree.
make sure the spaces between the cards are good.
check that the component card is being used correctly
